CoreFeatures/Checkpoint/CheckpointActor/src/test/java/info/smart_tools/smartactors/checkpoint/checkpoint_actor/CheckpointActorTest.java

```diff
@@ -292,7 +292,7 @@ public void Should_sendFeedbackMessage()
 
         verify(messageProcessingSequenceMock, never()).end();
 
-        verify(messageBusHandlerMock).handle(iObjectArgumentCaptor.capture(), eq("checkpoint_feedback_chain__0"), eq(true));
+        verify(messageBusHandlerMock).handle(iObjectArgumentCaptor.capture(), eq("checkpoint_feedback_chain"), eq(true));
```

Comment:
You probably should fix some broken code instead of failed assertion in test.

Reply:
Code isn't broken, just changed functionality. MessageBusHandler.handle method now receives chain name as parameter, instead of chain id before. What I really missed is that I did not delete 'chain_id_from_map_name_and_message' strategy registration in this test.